docs/plans/2024-12-30-whats-new-release-history-design.md

```
@@ -0,0 +1,196 @@
+
+```typescript
+interface WhatsNewVersionSectionProps {
+  release: WhatsNewRelease;
+  isNew: boolean; // Shows NEW badge + accent styling
+  defaultExpanded: boolean; // Initial expand state
+}
+```
+
+**Header layout:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)